db/migrate/20151217211019_create_spotlight_exhibit_filters.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSpotlightExhibitFilters < ActiveRecord::Migration
+  def change
+    create_table :spotlight_exhibit_filters do |t|
+      t.string :field
+      t.string :value
+      t.references :spotlight_exhibit, index: true, foreign_key: true
```

Comment:
Yeah, I was chasing a bug where Postgres wansn't generating the correct foreign_key constraint, but that may only exist when both models are not in the same namespace.  I'll try it the simplest way again.